src/app/settings/security-keys.component.html

```diff
@@ -0,0 +1,18 @@
+<app-change-kdf *ngIf="showChangeKdf"></app-change-kdf>
+<div
+  [ngClass]="{ 'tabbed-header': !showChangeKdf, 'secondary-header': showChangeKdf }"
+  class="border-0 mb-0"
+>
+  <h1>{{ "apiKey" | i18n }}</h1>
+</div>
+<p>
+  {{ "userApiKeyDesc" | i18n }}
+</p>
+<button type="button" class="btn btn-outline-secondary" (click)="viewUserApiKey()">
```

Comment:
Could you update all buttons to use the button directive from the component library? :)

```suggestion
<button bit-button type="button" (click)="viewUserApiKey()">
```
```

Reply:
Absolutely - will go through the modified components and update to the new buttons! Thanks for the reminder.

Reply:
Updated - left one todo regarding translation of bootstraps `btn-sm`

Reply:
@vincentsalucci could you speak with @danielleflinn regarding the `btn-sm` usages? If it's a pattern we want to keep we should add the `size` attribute to `bit-button`.

Reply:
Removed comment and started conversation with design. Will create jira task if needed.